#pragma once

#include <exception>

using std::exception;